packages/lodestar-cli/src/commands/beacon.ts

```diff
@@ -33,6 +43,11 @@ export class BeaconNodeCommand implements ICliCommand {
       .command("beacon")
       .description("Start lodestar node")
       .option("-c, --configFile [config_file]", "Config file path")
+      .option("-p, --preset [preset]", "Minimal/mainnet", "minimal")
+      .option("-s, --genesisState [params]", "Start chain from known state")
+      .option("--peer-id [peerId]", "peer id hex string or json file path")
+      .option("-e, --eth1", "interop for using fake eth1", "chain")
+      .option("-b, --eth1BlockNum", "eth1 block number to create genesis state")
```

Comment:
We should be watching the eth1 chain to determine the right eth1 block number

Reply:
Yeah I'll keep the watching code to look for the 1st eth1 block forming genesis state. Should we still keep this option too or not? 
After genesis time, if we want to launch a node we may need to specify the exact eth1 block number. Or in this case the genesis state of the new node is determined by `initial sync` to other peers?

Reply:
After genesis time, we should still be able to scan eth1 block by block in the same way to determine the genesis state. I'm in favor of removing this option.